google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/FirestoreImpl.java

```diff
@@ -339,9 +339,11 @@ public void onSuccess(List<WriteResult> writeResults) {
                             span.end();
                             resultFuture.set(userResult);
                           }
-                        });
```

Comment:
This should be `MoreExecutors.directExecutor()`

Reply:
All callbacks that didn't have an explicit executor should use `MoreExecutors.directExecutor()`